content/engine/security/rootless.md

```diff
@@ -390,7 +390,7 @@ For example:
 
 ### Errors when starting the Docker daemon
 
-**[rootlesskit:parent] error: failed to start the child: fork/exec /proc/self/exe: operation not permitted**
+**\[rootlesskit:parent\] error: failed to start the child: fork/exec /proc/self/exe: operation not permitted**
```

Comment:
Not for this PR, but  really wondering what the value is of the `[rootlesskit:parent]` in these; perhaps we should change these to the error-message alone (perhaps we should also remove the `error:`);

```
failed to start the child: fork/exec /proc/self/exe: operation not permitted
```